hooks/useWeatherData.ts

```diff
@@ -6,8 +6,9 @@ import {
   useGetForecastByCityQuery,
   useGetWeatherByCityQuery,
 } from "../services/api";
+import { DayWeatherSummary } from "../types/weatherTypes";
 
-export const useWeatherData = (city: string, today: boolean) => {
+export const useWeatherData = (city: string, today?: boolean) => {
```

Comment:
Ten hook robi wg mnie za dużo rzeczy. To było jeszcze jakoś do uzasadnienia, jak on po prostu przygotowywał wszystko co było potrzebne do wyświetlania tego SingleDayScreen. Ale teraz jak to jest używane też w AnimatedHeader i tym FiveDaysScreen trzeba może pomyśleć, czy to nie mogłoby się składać z jakichś mniejszych komponowalnych kawałków.

https://en.wikipedia.org/wiki/Single-responsibility_principle

Reply:
Tylko ze AnimatedHeader i FiveDaysScreen uzywaja w wiekszosci tych samych rzeczy co singleDayScreen, jedyna roznica to ze dodatkowo uzywaja getWeatherSummaryForDay i nie wiem jakbym mogl to podzielic zeby to bylo logiczne